packages/communication-react/stable-review/communication-react.api.md

```diff
@@ -993,13 +993,6 @@ export interface ChatMessage extends MessageCommon {
     // (undocumented)
     mine?: boolean;
     // (undocumented)
-    readBy?: {
```

Comment:
We will need to revert this, this breaks stable API

Reply:
Hmmm... looks like it is already in 1.1.0, as long as we don't need it anymore, maybe we could just leave the api, but not put anything into it

Reply:
I just revert this and still make it open to our customers, so for our customer they have 2 ways to render message readBy:
1. Provide an array of readReceipt for the whole messageThread, then the component will calculate each RR for each message and this is what we are doing (most optimized way)
2. Override RR for each single message by providing this property - not optimized in perf, but good to have this flexibility

Reply:
We could also mark the existing fields as deprecated and point them to the new location (but we'll need to get populating the fields until a breaking change)

Reply:
Actually wait, I'm getting very confused.
In our APIView for 1.1.0 we had them sign off on readBy and readNumber: see `export interface ChatMessage MessageCommon` in https://apiview.dev/Assemblies/Review/42fe51ace1e24184a42351ff89044352?revisionId=174cc81091854531884d2326cbc15d6c&doc=False

**However**, I do not see that in our API in our release version: https://github.com/Azure/communication-ui-library/blob/d523bb3380d88e3097ba73864b8f998c4e3db822/packages/communication-react/review/communication-react.api.md?plain=1#L811

Did we submit the wrong file to review somehow in 1.1.0?

Reply:
Hmmm... Look like so, I was a little confused by that as well cause we didn't have this feature in 1.1.0

Reply:
If this is the case, go ahead and remove this part @carocao-msft

Reply:
Let's remove this since this api wouldn't be needed in any following stable version, and let's grab a new quick approval from ARB team if necessary ( should be good cause we just take an api back, which doesn't actually exist in previous version)